### PR TITLE
Add 15 interactive examples

### PR DESCRIPTION
**Pull request by NiedziolkaMichal**
### Description

This PR adds 15 interactive examples and it also fixes the height of 3 examples added by [#22900](https://github.com/mdn/content/pull/22900)

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/generatorfunction/index.md [hunk 1]
```diff
@@ -18,11 +18,7 @@ The **`GeneratorFunction`** object provides methods for [generator functions](/e
 
 Note that `GeneratorFunction` is _not_ a global object. It can be obtained with the following code:
 
-```js
-const GeneratorFunction = function* () {}.constructor;
-```
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/generatorfunction/index.md:23]: I don't think this should be deleted. Makes the code slightly less discoverable.

Comment (NiedziolkaMichal) [reply files/en-us/web/javascript/reference/global_objects/generatorfunction/index.md]: Alright. I have added them back, under `Syntax` heading.

File: files/en-us/web/css/break-after/index.md [hunk 2]
```diff
@@ -56,10 +60,6 @@ To determine if a break must be done, the following rules are applied:
 
 Once forced breaks have been applied, soft breaks may be added if needed, but not on element boundaries that resolve in a corresponding `avoid` value.
 
-## Syntax
```
Comment (Josh-Cena) [inline files/en-us/web/css/break-after/index.md:59]: Are these syntax deletions intended?

File: files/en-us/web/javascript/reference/global_objects/asyncgeneratorfunction/index.md [hunk 3]
```diff
@@ -11,12 +11,14 @@ The **`AsyncGeneratorFunction`** object provides methods for [async generator fu
 
 Note that `AsyncGeneratorFunction` is _not_ a global object. It can be obtained with the following code:
 
+{{EmbedInteractiveExample("pages/js/async-functionasterisk-function.html", "taller")}}
+
+## Syntax
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/asyncgeneratorfunction/index.md:16]: This is not a syntax; just add the interactive example beneath the code block

File: files/en-us/web/javascript/reference/global_objects/generatorfunction/index.md [hunk 4]
```diff
@@ -18,12 +18,14 @@ The **`GeneratorFunction`** object provides methods for [generator functions](/e
 
 Note that `GeneratorFunction` is _not_ a global object. It can be obtained with the following code:
 
+{{EmbedInteractiveExample("pages/js/functionasterisk-function.html", "taller")}}
+
+## Syntax
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/generatorfunction/index.md:23]: Same here

**Review by Josh-Cena — COMMENTED** [review]
Sometimes the example is inserted in awkward places...

File: files/en-us/web/css/break-after/index.md (hunk 2 above)
Comment (NiedziolkaMichal) [reply files/en-us/web/css/break-after/index.md:59]: They were moved upwards. Take a look at https://github.com/mdn/content/pull/15132#pullrequestreview-946301084

File: files/en-us/web/javascript/reference/global_objects/asyncgeneratorfunction/index.md (hunk 3 above)
Comment (NiedziolkaMichal) [reply files/en-us/web/javascript/reference/global_objects/asyncgeneratorfunction/index.md]: Alright. I have removed `Syntax` and moved the code to its original place, so just `, "taller"` is added.

File: files/en-us/web/javascript/reference/global_objects/generatorfunction/index.md (hunk 4 above)
Comment (NiedziolkaMichal) [reply files/en-us/web/javascript/reference/global_objects/generatorfunction/index.md]: Alright. I have removed `Syntax` and moved the code to its original place, so just `, "taller"` is added.

**Review by Josh-Cena — APPROVED** [review]
Approving the JS changes, but I'm not sure about CSS. cc @estelle for review

**Comment by NiedziolkaMichal** [comment]
@Josh-Cena Ok, thank you

**Comment by Josh-Cena** [comment]
Friendly ping for @estelle again to look at the CSS structures

**Review by estelle — APPROVED** [review]

**Comment by NiedziolkaMichal** [comment]
Thanks!

**Comment by estelle** [comment]
thanks. And, @Josh-Cena, thanks for the ping!

**Pull request merged**